### PR TITLE
Product reviews

### PR DESCRIPTION
**Pull request by Ash-D23**
Added Product Reviews in Product Page with API Calls

reviews

### CODE REVIEW

File: src/Components/ProductReviews/ProductReviews.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+.review__input:focus{
+    outline: none;
+}
+
+.para-rating__container{
+    padding: 0.25rem 0.5rem;
+    font-size: 1.5rem;
+    background-color: green;
+    color: white;
```
Comment (znjs) [inline src/Components/ProductReviews/ProductReviews.css]: can use variables for colours
if possible try using different shades of the default colours, default colours are a little strong for eyes

File: src/Components/RatingsInput/RatingsInput.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        <label for="star4"></label>
+                
+        <input onClick={(e)=> setratings(e.target.value)} type="radio" name="stars" id="star3" value="3" checked={ratings==="3"} />
+        <label for="star3"></label>
+                
+        <input onClick={(e)=> setratings(e.target.value)} type="radio" name="stars" id="star2" value="2" checked={ratings==="2"} />
+        <label for="star2"></label>
+                
+        <input onClick={(e)=> setratings(e.target.value)} type="radio" name="stars" id="star1" value="1" checked={ratings==="1"} />
+        <label for="star1"></label>
```
Comment (znjs) [inline src/Components/RatingsInput/RatingsInput.js]: you can make an array and use a map over it, they are repeated elements

**Review by znjs — APPROVED** [review]
- Clean Code
- Good naming conventions used
- Good folder structure

**Review by manismk — APPROVED** [review]
Clean code and good naming convention.
Good to see adding more features

File: src/Components/ProductReviews/ProductReviews.css (hunk 1 above)
Comment (Ash-D23) [reply src/Components/ProductReviews/ProductReviews.css]: thanks, have updated the same

File: src/Components/RatingsInput/RatingsInput.js (hunk 2 above)
Comment (Ash-D23) [reply src/Components/RatingsInput/RatingsInput.js]: Added data in array and rendered

**Pull request merged**